client/actions/graphql/eventGQLActions.js

```diff
@@ -0,0 +1,83 @@
+import EVENT_LIST_GQL from '../../Graphql/Queries/EventListGQL';
+//import EVENT_GQL from '../../Graphql/Queries/EventGQL';
+import CREATE_EVENT_GQL from '../../Graphql/Mutations/CreateEventGQL';
+import UPDATE_EVENT_GQL from '../../Graphql/Mutations/UpdateEventGQL';
+import DEACTIVATE_EVENT_GQL from '../../Graphql/Mutations/DeactivateEventGQL';
+
+import { GET_EVENT, CREATE_EVENT, GET_EVENTS, UPDATE_EVENT, DEACTIVATE_EVENT } from '../constants';
+
+import { handleError } from '../../utils/errorHandler';
+import Client from '../../client';
+
+
+export const getEventsList = (before = '', after = '', first = 1, last = 1) => dispatch => Client.query(
+  EVENT_LIST_GQL(before, after, first, last)
+).then(data => dispatch({type: GET_EVENTS, payload: data.data, error: false}))
+.catch(error => handleError(error, dispatch));
+
+/**
+ * This commented out code below has a use case
+ * where one can not access events that they have deactivated,
+ * You would use this to view them, not sure how it fits in with where one would need
+ * to get the id of the event
+ */
+
+// export const getEvent = id => dispatch => Client.query(
```

Comment:
well done @malmike but please check line 25 to 28.

Reply:
What's the status here

Reply:
Removed them, handling the code climate issues then push

Reply:
@malmike is this ready for merge?